activerecord/test/schema/schema.rb

```diff
@@ -275,6 +275,7 @@
     t.index [:name, :description], length: 10
     t.index [:firm_id, :type, :rating], name: "company_index", length: { type: 10 }, order: { rating: :desc }
     t.index [:firm_id, :type], name: "company_partial_index", where: "(rating > 10)"
+    t.index [:firm_id, :type], name: "company_include_index", include: [:name, :account_id]
```

Comment:
There's a postgres specific schema in activerecord/test/schema/postgresql_specific_schema.rb which might be a more appropriate file for this change since this won't work on any other adapters.